webapp/i18n/en.json

```diff
@@ -105,6 +105,7 @@
   "dCb7CD": "Start call",
   "dmDHwc": "{count, plural, =1 {# recording} other {# recordings}} available",
   "eB0u7Z": "Calls are a quick, audio-first, way to interact with your team. Upgrade to Mattermost Professional to use calls in channels and group messages.",
+  "eP+xGA": "Disabled because the <code>RTCD service URL</code> field is non-empty.",
```

Comment:
Some non-blocking thoughts:

- The `<code>` formatting seems a bit strong on the eye to me. Could we switch to italic maybe? That would be consistent with how we mention fields in our documentation I believe.
- What do you think about rephrasing to something like `Disabled as not applicable when the RTCD service URL field is non-empty.`?

Reply:
- Ok, changed to italic, but I personally don't like it as much as the code formatting. But it's not a big deal for me, so whatever you prefer is great.
- rephrased to: `Not applicable when the RTCD service URL field is in use.` which is a bit clearer imo. Let me know?

![CleanShot 2023-05-24 at 15 49 03@2x](https://github.com/mattermost/mattermost-plugin-calls/assets/1490756/39fdb9b0-0550-4e54-8fcf-4a294235133b)

Reply:
> * Ok, changed to italic, but I personally don't like it as much as the code formatting. But it's not a big deal for me, so whatever you prefer is great.

Fair enough. What about a link to https://docs.mattermost.com/configure/plugins-configuration-settings.html#rtcd-service-url? 

> * rephrased to: `Not applicable when the RTCD service URL field is in use.` which is a bit clearer imo. Let me know?

Yep, looks great.

Reply:
Yep, that looks fine to me: 

![CleanShot 2023-05-24 at 15 57 36@2x](https://github.com/mattermost/mattermost-plugin-calls/assets/1490756/f612e584-e5e9-4f82-ab73-962e3dec6205)

Reply:
:100: